src/app.jsx

```diff
@@ -20,12 +20,21 @@ function showApp() {
   `;
   document.body.appendChild(root);
 
+  const RedirectToClearSky = () => {
+    if (window.location.href.includes('bsky.thieflord.dev')) {
+      window.location.href = 'https://clearsky.app';
+      return null;
+    }
+    return null;
+  };
+
   const router = createBrowserRouter([
     { path: '/', element: <Home /> },
     { path: '/index.html', element: <Home /> },
     { path: '/stable/*', element: <Home /> },
     { path: '/:handle', element: <AccountView /> },
     { path: '/:handle/:tab', element: <AccountView /> },
+    { path: '*', element: <RedirectToClearSky /> },
```

Comment:
i think this splat route will only ever get rendered for paths with 3 or more segments any with 2 or less will match one of the above first

Reply:
@noahm This may work a little different since moving to the new hosting for the frontend but, the old hosting said that the redirect would have to be in app as well as a DNS redirect. I added the CNAME record to point to clearsky.app not too long ago and it isn't redirecting yet.

Reply:
for context, this is how it was being redirected in the python server:

```@app.before_request
def redirect_to_clearsky() -> quart.redirect:
    if request.host == 'bsky.thieflord.dev':
        logger.debug("Redirecting to clearsky.app")
        redirect_url = request.url.replace('bsky.thieflord.dev', 'clearsky.app', 1)

        return redirect(redirect_url, code=301)```